session_test.go

```diff
@@ -981,13 +991,7 @@ func TestSession_PingOfDeath(t *testing.T) {
 }
 
 func TestSession_ConnectionWriteTimeout(t *testing.T) {
-	// Disable keepalives so they don't detect the failed connection
```

Comment:
Heh; I completely forgot the whole reason I was reworking the tests was so I could add my new test case. Sorry about that… Shortly, yes.